app/models/saved_url.rb

```diff
@@ -0,0 +1,2 @@
+class SavedUrl < ApplicationRecord
```

Comment:
I would expect that this class to use a polymorphic relationship like we do with the `AdvancedSetting` model: https://github.com/ManageIQ/manageiq/blob/master/app/models/advanced_setting.rb#L2

Then the mixin would expose a `has_one` or `has_many` like we do for the Host model: https://github.com/ManageIQ/manageiq/blob/master/app/models/host.rb#L86

Also, would this be better named as something like `ExternalUrl` or `CustomUrl`?